### ISSUE TITLE
Mozilla Firefox Support

### ISSUE THREAD

**Issue by DillonB07** [lafkpages/XL-Replit#10]
Support for the extension on Mozilla Firefox would be great!

I will work on this and make a PR when it is done.

**Comment by lafkpages** [comment]
Does Firefox support Chrome extensions without much change in code? Or would we have to make this twice, once for each browser?

**Comment by DillonB07** [comment]
Very similar, you should be able to use the same codebase.

**Comment by Tulip-0333** [comment]
Oh so that's the reason I'm not able to use this, I'm on Firefox!

**Comment by DillonB07** [comment]
Yep, you need a Chromium based browser currently. Hopefully I’ll get around to making it for Firefox in the next few weeks, but it’s low priority, I’ve got a lot of other things I need to do first!

**Comment by lafkpages** [comment]
I tried loading this extension on Firefox and it worked with no changes in my code. I have merged no PRs for Firefox support, and I'm still using `chrome` and it works perfectly on Firefox. The only thing I had to change for full Firefox support was to not use the `:has()` selector, as it's not enabled by default on Firefox.

**Comment by DillonB07** [comment]
@LuisAFK what version of FireFox are you using? When I tried to use the extension on FF with no changes, the popup was really wide and none of the functionality worked. This wasn’t due to my settings as I tried it in a private browsing window.

**Comment by lafkpages** [comment]
As we saw in VC, you have to give the extension permissions to "Always allow on replit.com". My Firefox version is `111.0.1 (64-bit)`.
